src/main.rs

```diff
@@ -62,6 +64,11 @@ fn main() {
             println!("hermit git is not yet implemented");
         }
         ("init", Some(matches)) => {
+            let name = match matches.value_of("SHELL_NAME") {
+                Some(shell_name) => shell_name,
+                None => "default",
+            };
+            init::new_profile(name);
```

Comment:
The more I think about it, I think the matches should be passed in to a top-level function in each module that does this sort of argument extraction.